src/daft-plan/src/logical_optimization/rules/split_actor_pool_projects.rs

```diff
@@ -715,75 +715,6 @@ mod tests {
         Ok(())
     }
 
-    #[cfg(not(feature = "python"))]
-    #[test]
-    fn test_multiple_with_column_parallel_common_subtree_eliminated() -> DaftResult<()> {
-        let scan_op = dummy_scan_operator(vec![Field::new("a", daft_core::DataType::Utf8)]);
-        let scan_plan = dummy_scan_node(scan_op);
-        let stateful_project_expr = create_stateful_udf(vec![col("a")]);
-
-        // Add a Projection with StatefulUDF and resource request
-        // NOTE: Our common-subtree elimination will build this as 2 project nodes:
-        // Project([col("a").alias("a"), foo(col("a")).alias(factored_column_name)])
-        //   --> Project([col("a"), col(factored_column_name).alias("b"), col(factored_column_name).alias("c")])
-        let factored_column_name = "Function_Python(Stateful(StatefulPythonUDF { name: \"foo\", num_expressions: 1, return_dtype: Int64, resource_request: Some(ResourceRequest { num_cpus: Some(8.0), num_gpus: Some(1.0), memory_bytes: None }), batch_size: None, concurrency: Some(8) }))(a)";
```

Comment:
I removed this test because it keeps failing everytime I change the representation of the struct, because of how we construct the column name. It's super annoying and makes refactoring very difficult.